python/ray/tune/tests/test_sample.py

```diff
@@ -982,6 +982,32 @@ def testConvertHyperOpt(self):
         self.assertTrue(5 <= config["a"] <= 6)
         self.assertTrue(8 <= config["b"] <= 9)
 
+    def testConvertHyperOptChooseFromListOfList(self):
+        from ray.tune.suggest.hyperopt import HyperOptSearch
+        from hyperopt import hp
+
+        config = {
+            "a": tune.choice([[1, 2], [3, 4]]),
+        }
+        converted_config = HyperOptSearch.convert_search_space(config)
+        hyperopt_config = {
+            "a": hp.choice("a", [[1, 2], [3, 4]]),
+        }
+
+        searcher1 = HyperOptSearch(
+            space=converted_config, random_state_seed=1234, metric="a", mode="max"
+        )
+        searcher2 = HyperOptSearch(
+            space=hyperopt_config, random_state_seed=1234, metric="a", mode="max"
+        )
+
+        config1 = searcher1.suggest("0")
+        config2 = searcher2.suggest("0")
+
+        self.assertEqual(config1, config2)
+
+        self.assertTrue(config1.get("a") in [(1, 2), (3, 4)])
```

Comment:
Just to clarify, this means that hyperopt converts into tuples even though we specify lists, right?

Reply:
Yes, correct.

Reply:
Let's add a comment for it!

Reply:
That's a good idea. Added it.